gui/default/index.html

```diff
@@ -570,7 +570,7 @@ <h4 class="panel-title">
             <button type="button" class="btn btn-sm btn-default" ng-click="setAllFoldersPause(false)" ng-if="isAtleastOneFolderPausedStateSetTo(true)">
               <span class="fas fa-play"></span>&nbsp;<span translate>Resume All</span>
             </button>
-            <button type="button" class="btn btn-sm btn-default" ng-click="rescanAllFolders()" ng-disabled="!isAtleastOneFolderPausedStateSetTo(false)">
+            <button type="button" class="btn btn-sm btn-default" ng-click="rescanAllFolders()" ng-if="isAtleastOneFolderPausedStateSetTo(true) || isAtleastOneFolderPausedStateSetTo(false)" ng-disabled="!isAtleastOneFolderPausedStateSetTo(false)">
```

Comment:
It seems like there ought to exist an easier criteria here, like the set of folders actually being non-empty, instead of iterating twice over the folders that do exist...

Reply:
Yeah, I know that this is hackish, but I could not come up with anything better using the current code. My JS skills are kind of limited, so I may have very likely missed something obvious though.

Reply:
We have an `isEmptyObject` function so I think something like `ng-if="!isEmptyObject(folders)"` should do it

Reply:
I could not figure out the `isEmptyObject` method, but I have analysed the rest of the code, and in the end I have taken a different approach with `.length > 0` (which is already used in many other places in `index.html`.). I hope that this is fine now.